resin_calculator/artifact_calc.py

```diff
@@ -15,6 +15,9 @@
     "goblet": ["hp%", "def%", "atk%", "elemental mastery", "elemental damage", "physical damage"]
 }
 
+#Not sure if valid substats should be placed in the sub stat validation function, or if I should leave as is.
```

Comment:
I sort of like these things being global. It makes the functions less cluttered, and makes them easily accessible in case you wind up needing them elsewhere, including in tests.

For consistency, I would maybe also move the set lists up here. That puts all of your built-in data in one spot.

Reply:
Relocated!

Reply:
Now that things are global I recommend indicating in the variable name in some way that they are global. PEP8 standard calls out for full capitalization so I would recommend that.

Reply:
Oh, I wasn't aware there was a different naming convention, but I really like that!

Reply:
It certainly looks better this way. Change was made!